app/javascript/packs/idv-phone-alert.ts

```diff
@@ -5,8 +5,6 @@ const { iti, textInput: input } = document.querySelector('lg-phone-input') as Ph
 const failedPhoneNumbers: string[] = JSON.parse(alertElement.dataset.failedPhoneNumbers!);
 
 input.addEventListener('input', () => {
-  const isFailedPhoneNumber = failedPhoneNumbers.includes(
-    iti.getNumber(intlTelInputUtils.numberFormat.E164),
-  );
+  const isFailedPhoneNumber = failedPhoneNumbers.includes(iti.getNumber());
```

Comment:
This change is made because `intl-tel-input` no longer assigns a global `intlTelInputUtils`, and E.164 is the default ([source](https://github.com/jackocnr/intl-tel-input#:~:text=return%20%221234%22-,getNumber,-Get%20the%20current)).